node/src/server-modules/GlideFtOptions.ts

```diff
@@ -118,3 +118,39 @@ export interface FtCreateOptions {
     /** The prefix of the key to be indexed. */
     prefixes?: GlideString[];
 }
+
+/**
+ * Represents the input options to be used in the FT.SEARCH command.
+ * All fields in this class are optional inputs for FT.SEARCH.
+ */
+export interface FtSearchOptions {
+    /**
+     * Add a field to be returned.
+     * @param fieldIdentifier field name to return.
+     * @param alias optional alias for the field name to return.
+     */
+    returnFields?: { fieldIdentifier: GlideString; alias?: GlideString }[];
+
+    /** Query timeout in milliseconds. */
+    timeout?: number;
+
+    /**
+     * Query parameters, which could be referenced in the query by `$` sign, followed by
+     * the parameter name.
+     */
+    params?: GlideRecord<GlideString>;
+
+    /**
+     * Configure query pagination. By default only first 10 documents are returned.
+     *
+     * @param offset Zero-based offset.
+     * @param count Number of elements to return.
+     */
+    limit?: { offset: number; count: number };
+
+    /**
+     * Once set, the query will return only the number of documents in the result set without actually
+     * returning them.
+     */
+    count?: boolean;
```

Comment:
You can make limit and count mutually exclusive there

Reply:
...I didn't try it, does it give you an error if both are given?

Reply:
As I understand the docs - yes

Reply:
will write a test then...

Reply:
it does not throw an error.  we could maybe include this detail in the documentation (that count will override the limit) but that's a detail that may change.  I dislike getting into such detail about the services' logic when their API is unclear.